pulp_rpm/app/fields.py

```diff
@@ -0,0 +1,57 @@
+from rest_framework import serializers
+from pulp_rpm.app.models import UpdateCollection, UpdateReference
+
+
+class UpdateCollectionPackageField(serializers.ListField):
```

Comment:
This field is to represent all the collections (which contain packages).
The class name is confusing, because it seems like it deals with packages and not with collections themselves, because we have UpdateCollection and UpdateCollectionPackage models.
I suggest to call it UpdateCollectionField, since it deals with collections and everything which they contain.

Reply:
Name changed to more self-explained 'UpdateCollectionField'